integration/simulation/simulation_in_mem_test.go

```diff
@@ -24,11 +42,75 @@ func TestInMemoryMonteCarloSimulation(t *testing.T) {
 		SimulationTime:            15 * time.Second,
 		L1EfficiencyThreshold:     0.2,
 		L2EfficiencyThreshold:     0.32,
-		L2ToL1EfficiencyThreshold: 0.34,
+		L2ToL1EfficiencyThreshold: 0.5,
+		TxHandler:                 ethereummock.NewMockTxHandler(),
 	}
 
 	simParams.AvgNetworkLatency = simParams.AvgBlockDuration / 15
 	simParams.AvgGossipPeriod = simParams.AvgBlockDuration * 2 / 7
 
 	testSimulation(t, network.NewBasicNetworkOfInMemoryNodes(), simParams)
 }
+
+// TestMemObscuroRealEthMonteCarloSimulation runs the simulation against a ganache network
+// 1. Install ganache -> npm install ganache --global
+// 2. Run ganache -> rm -rf ganachedb && ganache --database.dbPath="./ganachedb" -l 1024000000000 --wallet.accounts="0x5dbbff1b5ff19f1ad6ea656433be35f6846e890b3f3ec6ef2b2e2137a8cab4ae,0x56BC75E2D63100000" --wallet.accounts="0xb728cd9a9f54cede03a82fc189eab4830a612703d48b7ef43ceed2cbad1a06c7,0x56BC75E2D63100000" -p 7545 -g 225
+func TestMemObscuroRealEthMonteCarloSimulation(t *testing.T) {
+	t.Skip("test under construction")
+	setupTestLog()
+
+	// private key is prefunded and used to issue txs - used here to deploy contract ahead of node initialization
+	tmpWallet := wallet.NewInMemoryWallet("5dbbff1b5ff19f1ad6ea656433be35f6846e890b3f3ec6ef2b2e2137a8cab4ae")
+	contractAddr := deployContract(t, tmpWallet)
+
+	params := params.SimParams{
+		NumberOfNodes:             2,
+		NumberOfWallets:           2,
+		AvgBlockDuration:          10 * time.Millisecond,
+		SimulationTime:            15 * time.Second,
+		L1EfficiencyThreshold:     0.9, // todo review this
```

Comment:
these need to be reviewed as well

Reply:
Set them as close as I could to the in-memory efficiencies 👍